apps/fulfillment-e2e/src/integration/partialy-picking-picklist.cy.ts

```diff
@@ -0,0 +1,158 @@
+const pickingFragment = new PickingFragment();
+
+describe('Partial picking a picklist', () => {
+  beforeEach(() => {
+    cy.clearIndexedDB();
+    cy.login();
+    pickingPage.visit();
+  });
+
+  it('should check partial picking', () => {
+    // Setting initial number of products in each tab
+    pickingPage.insideTabContent(pickingPage.getNotPickedTab(), () => {
+      pickingProductFragment.getProducts().then((products) => {
+        cy.wrap(products.length).as('initialNotPickedListItemsNumber');
+      });
+    });
+    pickingPage.insideTabContent(pickingPage.getPickedTab(), () => {
+      pickingProductFragment.getProducts().should('not.exist');
+      cy.wrap(0).as('initialPickedListItemsNumber');
+    });
+    pickingPage.insideTabContent(pickingPage.getNotFoundTab(), () => {
+      pickingProductFragment.getProducts().should('not.exist');
+      cy.wrap(0).as('initialNotFoundListItemsNumber');
+    });
+
+    // For the first item, select only part of the quantity.
+    pickingPage.insideTabContent(pickingPage.getNotPickedTab(), () => {
+      pickingProductFragment
+        .getProducts()
+        .eq(0)
+        .within(() => {
+          pickingProductFragment.getQuantityInputPlusButton().click();
+          pickingProductFragment.getSubmitButton().click();
+        });
+    });
+
+    // Verify that the Partial Picking Modal is opened.
+    pickingFragment.getPartialPickingDialog().should('have.attr', 'open');
+
+    // Close dialog
+    pickingFragment.getPartialPickingCancelButton().click();
+
+    // Dialog should be closed
+    pickingFragment.getPartialPickingDialog().should('not.have.attr', 'open');
+
+    // Not Picked tab list should have the same number of products
+    cy.get('@initialNotPickedListItemsNumber').then((initCount) => {
+      pickingPage.checkTabProductsCount(
+        pickingPage.getNotPickedTab(),
+        Number(initCount)
+      );
+    });
+
+    // Open modal again
+    pickingPage.insideTabContent(pickingPage.getNotPickedTab(), () => {
+      pickingProductFragment
+        .getProducts()
+        .eq(0)
+        .within(() => {
+          pickingProductFragment.getSubmitButton().click();
+        });
+    });
+
+    // Verify that the Partial Picking Modal is opened.
+    pickingFragment.getPartialPickingDialog().should('have.attr', 'open');
+
+    // Confirm partial picking
+    pickingFragment.getPartialPickingConfirmButton().click();
+
+    // Dialog should be closed
+    pickingFragment.getPartialPickingDialog().should('not.have.attr', 'open');
+
+    // Verify that the picking item is moved to the "Picked" tab with the entered quantity, and the "Not found" tab with the not entered quantity.
+    // Not Picked tab list should have x-2 products
+    cy.get('@initialNotPickedListItemsNumber').then((initCount) => {
+      pickingPage.checkTabProductsCount(
+        pickingPage.getNotPickedTab(),
+        Number(initCount) - 1
+      );
+    });
```

Comment:
If this - https://github.com/spryker/oryx/pull/290/files#r1177574104 - is done, this code can be simplified to 

```
cy.get('@initialNotPickedCount').then((initCount) => {
  pickingPage.getNumberOfNotPickedProducts().should('be.eq', initCount)
});
```